tinyfont.go

```diff
@@ -120,6 +120,6 @@ func LineWidth(f Fonter, str string) (innerWidth uint32, outboxWidth uint32) {
 }
 
 // GetGlyph returns the glyph corresponding to the specified rune in the font.
-func GetGlyph(f Fonter, r rune) Glyph {
+func GetGlyph(f Fonter, r rune) Glypher {
```

Comment:
This will revert the changes made in 05eb156cff8b312112ba7e43e414d7072da89950. The only cases where this change could cause issues would be if this function has been used after 05eb156cff8b312112ba7e43e414d7072da89950, but I believe such instances are rare.